include/engine/search_engine_data.hpp

```diff
@@ -75,12 +79,8 @@ template <> struct SearchEngineData<routing_algorithms::mld::Algorithm>
 
     static SearchEngineHeapPtr forward_heap_1;
     static SearchEngineHeapPtr reverse_heap_1;
-    static SearchEngineHeapPtr forward_heap_2;
```

Comment:
Just a heads up that @daniel-j-h will probably need at least two heaps for the alternative plugin down the road.